admin-ui/src/Meals/MealList.tsx

```diff
@@ -13,74 +13,87 @@ import {
   TabbedShowLayout,
   TextField,
   UrlField,
-} from "react-admin";
-import { ListField } from "../ListField";
-import { NutritionShow } from "../Nutrition/NutritionShow";
+} from 'react-admin';
```

Comment:
I am not sure why you have changed all the double quotes as single quotes. JSX attributes should be double quotes only. I see even they are changed to single.